tsak 10 css (mob.tab.desk) + html

Shurachok opened the pull request:
зробив css для мобільного, планшету, монітору.
також змінив код в html для коректногої підгрузки img

Shurachok committed:
tsak 10 css (mob.tab.desk) + html

зробив css для мобільного, планшету, монітору.
також змінив код в html для коректногої підгрузки img

Yurii-Ilkevych commented on src/css/task-ten.css:
```diff
@@ -65,10 +72,10 @@
 .rest-box-middle {
-  width: 100%;
+  width: 39vw;
   margin-top: 102px;
```
неправильно стоїть текст для мобільної версії занадто низько так як вище маємо клас rest-row-middle
    margin-top: 32px
то вже відступає текст на 32 px а загальний відступ є 102 але маємо від попереднього 32 тоді 102-32=70px
повинно бути 70

Yurii-Ilkevych replied:
а також не бачу тіней на мобільну версію

Shurachok committed:
bug corrected (css)

виправив правильне позіціювання для фото нижніх в секції (для мобільної версії)

Shurachok committed:
task 10 (shadows + text margin mob)b +
